src/main/java/org/spongepowered/common/text/chat/ChatUtil.java

```diff
@@ -47,18 +44,15 @@
     private ChatUtil() {
     }
 
-    public static void sendMessage(ITextComponent component, MessageChannel channel, CommandSource source, boolean isChat) {
-        final Text raw = SpongeTexts.toText(component);
-        final MessageFormatter formatter = new MessageEvent.MessageFormatter(raw);
-        final Cause cause = SpongeImplHooks.onServerThread() ? Sponge.getCauseStackManager().getCurrentCause() : Cause.of(EventContext.empty(), source);
+    public static void sendMessage(TextComponent component, Audience channel, CommandSource source, boolean isChat) {
+        final Component raw = TextComponent.builder(component.content()).build();
```

Comment:
The `component` parameter should still be a Minecraft `ITextComponent`. You can convert it using the `SpongeAdventure` class.